astropy/cosmology/io/ecsv.py

```diff
@@ -0,0 +1,93 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""Reader and Writer for `~astropy.cosmology.Cosmology` in the ECSV format.
+
+This module is NOT considered public API.
+
+"""
+
+import inspect
+import os
+
+from astropy.io import registry as io_registry
+from astropy.cosmology import Cosmology
+from astropy.table import QTable
+from astropy.utils.compat.optional_deps import HAS_YAML
+
+from .core import from_table, to_table
+
+
+__all__ = ["read_ecsv", "write_ecsv", "ecsv_identify"]
+
+
+def read_ecsv(*args, index=None, move_to_meta=False, **kwargs):
```

Comment:
No docstring for `move_to_meta`.

Reply:
Oops. Thanks. Re-reading it, the relevant docstring in ``from_mapping`` is a bit too technical. I'll clarify it.